dolphinscheduler-ui-next/src/views/projects/workflow/instance/use-table.ts

```diff
@@ -95,45 +93,7 @@ export function useTable() {
         title: t('project.workflow.status'),
         key: 'state',
         className: 'workflow-status',
-        render: (_row: IWorkflowInstance) => {
-          const stateIcon = taskStateIcon[_row.state]
-          const iconElement = h(
-            NIcon,
-            {
-              size: '18px',
-              style: 'position: relative; top: 7.5px; left: 7.5px',
-              class: stateIcon.classNames
```

Comment:
The ```WorkflowE2ETest``` using class ```success``` to verify the process instance success state. Remove it would lead to the ```WorkflowE2ETest``` failed.

Reply:
I will add the class by the task type.Such as SUBMITTED_SUCCESS will be submitted_success expect FAILURE which corresponds to failed.